doc/reference_management.xml

```diff
@@ -351,13 +349,57 @@ DROP TABLE my_schema.my_spatial_table;
 	  </refsection>
 	</refentry>
 
+	<refentry id="Find_SRID">
+	  <refnamediv>
+		<refname>Find_SRID</refname>
+
+		<refpurpose>Returns the SRID defined for a geometry column.</refpurpose>
+	  </refnamediv>
+
+	  <refsynopsisdiv>
+		<funcsynopsis>
+		  <funcprototype>
+			<funcdef>integer <function>Find_SRID</function></funcdef>
+			<paramdef><type>varchar </type> <parameter>a_schema_name</parameter></paramdef>
+			<paramdef><type>varchar </type> <parameter>a_table_name</parameter></paramdef>
+			 <paramdef><type>varchar </type> <parameter>a_geomfield_name</parameter></paramdef>
+		  </funcprototype>
+		</funcsynopsis>
+	  </refsynopsisdiv>
+
+	  <refsection>
+		<title>Description</title>
+
+		<para>Returns the integer SRID of the
+			specified geometry column by searching through the GEOMETRY_COLUMNS table.
+			If the geometry column has not been properly added (e.g. with the
+			<xref linkend="AddGeometryColumn" /> function), this function will not work.</para>
```

Comment:
AddGeometryColumn is not a good way to add columns since PostGIS 2?

Reply:
That's my understanding too.  So this description should indicate this more strongly.